doc/manual/plugins.rst

```diff
@@ -46,7 +50,6 @@ Global plugins
    plugins_global/errors
    plugins_global/rc
    plugins_global/wcsmatch
-   plugins_global/changehistory
```

Comment:
> extracting the string and writing it to a file

Do you have an idea on how to do this in Sphinx without defeating the purpose of Sphinx? Currently, I can retain the existing `changehistory.rst` file and insert `.. automodapi::` directive to pull in docstring into that file, but then I run into the section title problem described above.